### PR TITLE
qa: fs/upgrade test fixes and cephfs feature bit updates for Octopus/Nautilus

### PR DESCRIPTION
**Pull request by batrick**

### CODE REVIEW

File: src/mds/cephfs_features.h [hunk 1]
```diff
@@ -27,6 +27,8 @@
 #define CEPHFS_FEATURE_RECLAIM_CLIENT	10
 #define CEPHFS_FEATURE_LAZY_CAP_WANTED  11
 #define CEPHFS_FEATURE_MULTI_RECONNECT  12
+/* N.B.: oops, we can't use CEPHFS_FEATURE_NAUTILUS because we didn't add a bit for it. */
+#define CEPHFS_FEATURE_NAUTILUS         Ooops
```
Comment (ukernel) [inline src/mds/cephfs_features.h]: Oops?

File: src/mds/cephfs_features.h [hunk 2]
```diff
@@ -33,6 +33,7 @@
 #define CEPHFS_FEATURE_MULTI_RECONNECT  12
 /* N.B.: oops, we can't use CEPHFS_FEATURE_NAUTILUS because we didn't add a bit for it. */
```
Comment (ukernel) [inline src/mds/cephfs_features.h]: maybe:

#define CEPHFS_FEATURE_NAUTILUS   CEPHFS_FEATURE_MULTI_RECONNECT

File: src/mds/cephfs_features.h (hunk 1 above)
Comment (batrick) [reply src/mds/cephfs_features.h]: "Oops" is just a placeholder since it cannot be used.

File: src/mds/cephfs_features.h (hunk 2 above)
Comment (batrick) [reply src/mds/cephfs_features.h]: My interpretation of the extra bit for a release is that the client has code "up-to-date" with the rest of Ceph, not just the features CephFS cares about. So it's not really appropriate to alias to another feature bit. What do you think?

Comment (ukernel) [reply src/mds/cephfs_features.h]: I added CEPHFS_FEATURE_MULTI_RECONNECT support for ceph-fuse and kernel about the same time. when ceph-fuse and kernel client support CEPHFS_FEATURE_MULTI_RECONNECT, they also support CEPHFS_FEATURE_REPLY_ENCODING and CEPHFS_FEATURE_LAZY_CAP_WANTED. So it's good enough to rename CEPHFS_FEATURE_MULTI_RECONNECT to CEPHFS_FEATURE_NAUTILUS

Comment (batrick) [reply src/mds/cephfs_features.h]: Okay, I changed it.

**Comment by ukernel** [comment]
LGTM

**Comment by batrick** [comment]
Please approve then @ukernel (github checks prevent merges without one).

**Comment by batrick** [comment]
`Reviewed-by: Zheng Yan <zyan@redhat.com>`

**Comment by batrick** [comment]
(I forced the merge by editing GitHub protection rules.)

**Pull request merged**